spec/acceptance/init_spec.rb

```diff
@@ -5,57 +5,41 @@
   include Beaker::TaskHelper::Inventory
   include BoltSpec::Run
 
-  def module_path
-    RSpec.configuration.module_path
-  end
-
-  def config
-    { 'modulepath' => module_path }
-  end
-
-  def inventory
-    hosts_to_inventory.merge('features' => ['puppet-agent'])
-  end
-
-  def run(params)
-    run_task('service', 'default', params, config: config, inventory: inventory)
-  end
-
-  osfamily_fact = fact('osfamily')
+  osfamily_fact = os[:family]
 
   package_to_use = ''
   before(:all) do
     if osfamily_fact != 'windows'
```

Comment:
May as well be explicit with this and just use `os[:family]` instead of a variable since it's only a few occurrences.